### PR TITLE
Adnan's Discussions Functionality

### PR DESCRIPTION
**Pull request by adnanhemani**
Some work will need to be done on the UI of the discussion, but all functionality is accounted for.

### CODE REVIEW

File: app/controllers/session_controller.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def create
     user = User.user_from_oauth(oauth_hash)
     admin = Admin.admin_from_oauth(oauth_hash)
     session[:user_email] = oauth_hash[:info][:email]
+    #byebug
```
Comment (jjlgao) [inline app/controllers/session_controller.rb]: Maybe not leave this commented in master

File: app/models/discussion.rb [hunk 2]
```diff
@@ -29,5 +31,9 @@ def self.valid_discs_for(team)
         end
         return discs
     end
+    
+    def toString
+       return self.number.to_s + " " + self.day + " " + self.time
```
Comment (jjlgao) [inline app/models/discussion.rb]: Is it cleaner to write `return "#{self.number} #{self.day} #{self.time}"`?

File: app/views/discussion/add.html.erb [hunk 3]
```diff
@@ -2,7 +2,11 @@
 <body>
   <title>Add New Discussion</title>
     <div class="container">
-       
+        <% if @disc.errors.any? %>
+            <div class="error_messages alert alert-dismissible alert-danger">
+                CCN is either already used or is not 5 digits. Please check your inputs and try again.
```
Comment (jjlgao) [inline app/views/discussion/add.html.erb]: Why is this repeated between here and the notice above? Does this duplicate the error message, and are both necessary?

File: app/views/discussion/edit_index.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<html>
+<body>
+  <title>Edit Discussion Sections</title>
+  
+    <div class="container">
+      
+        <h1>Select Discussion Section To Edit</h1>
+        <%= form_tag :html => {:class => "form-horizontal"}, :url => new_discussion_path, :method => :get %>
+            <fieldset>
+            <div class="field form-group">
```
Comment (jjlgao) [inline app/views/discussion/edit_index.html.erb]: nit: tab this pls

File: config/environments/development.rb [hunk 5]
```diff
@@ -60,9 +60,9 @@
     port: 587,
     domain: 'gmail.com',
     user_name: 'enrollmeberkeley@gmail.com',
-    password: 'eecseecseecs',
+    password: 'xsuexspmvwbyifht',
```
Comment (jjlgao) [inline config/environments/development.rb]: shouldn't the other PR's stuff not be in this PR too?

File: app/controllers/session_controller.rb (hunk 1 above)
Comment (adnanhemani) [inline app/controllers/session_controller.rb]: Done

File: app/models/discussion.rb (hunk 2 above)
Comment (adnanhemani) [inline app/models/discussion.rb]: Changed to this

File: app/views/discussion/add.html.erb (hunk 3 above)
Comment (adnanhemani) [inline app/views/discussion/add.html.erb]: Sorry, notice wasn't needed. Fixed.

File: app/views/discussion/edit_index.html.erb (hunk 4 above)
Comment (adnanhemani) [inline app/views/discussion/edit_index.html.erb]: Fixed.

File: config/environments/development.rb (hunk 5 above)
Comment (adnanhemani) [inline config/environments/development.rb]: Sorry, I forgot to branch before starting the past pr.

**Comment by jjlgao** [comment]
lgtm except that you should delete the other PR if you merge this one, or remove the other PR's stuff

**Pull request merged**